pom.xml

```diff
@@ -64,23 +64,48 @@
     <dependency>
       <groupId>org.jenkins-ci.plugins.workflow</groupId>
       <artifactId>workflow-step-api</artifactId>
-      <version>2.3</version>
+      <version>2.12</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>script-security</artifactId>
-      <version>1.21</version>
+      <version>1.30</version>
+    </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins.workflow</groupId>
+      <artifactId>workflow-multibranch</artifactId>
```

Comment:
I'm not sure if it's such good idea to pull in these additional in. Any way around this? @jglick maybe? Sorry, my first Jenkins contribution.